### DISCUSSION TITLE
Demo of USAGE fails

### DISCUSSION THREAD

**Discussion by elemariosClearstar** [mganss/HtmlSanitizer#378] [General]
The demo of usage fails with version 0.17.1
![image](https://user-images.githubusercontent.com/105004960/181835376-68eb4b22-f3f1-4112-9b73-4c13632a37f6.png)
![image](https://user-images.githubusercontent.com/105004960/181836371-46e44d36-7b60-4019-88ca-92c006acd2d7.png)

**Comment by elemariosClearstar** [comment]
Sorry the version would be 7.1.542

**Comment by mganss** [comment]
Thanks, fixed.
